docker/fame/run.web.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+utils/run.sh utils/install_docker.py
+
+echo "[+] Ensuring presence of uwsgi"
+utils/run.sh -m pip install uwsgi > /dev/null
+
+TIMEOUT=60
+
+echo "[+] Waiting $TIMEOUT seconds for MongoDB to come up"
+python docker/wait-for.py fame-mongo 27017 $TIMEOUT
+if [ "$?" -ne "0" ]; then
+    echo "[X] Could not connect to MongoDB instance - is it up and running?"
+    exit 1
+fi
+
+echo "[+] Running webserver"
+chown fame:fame /fame -R
```

Comment:
This step should not be in the Dockerfile in my opinion, because there are several volumes that are mounted beneath /fame that may have the wrong permissions for FAME to access it. Thus, we do the `chown` at runtime.